engine/src/main/java/com/redhat/cloud/notifications/events/EndpointProcessor.java

```diff
@@ -113,7 +121,11 @@ public void process(Event event) {
                             }
                             break;
                         case EMAIL_SUBSCRIPTION:
-                            emailProcessor.process(event, endpointsByTypeEntry.getValue());
+                            //if (this.featureFlipper.isNotificationsEmailConnectorEnabled()) {
+                            //emailProcessor.process(event, endpointsByTypeEntry.getValue());
+                            //} else {
+                            emailProcessor2.process(event, endpointsByTypeEntry.getValue());
+                            //}
```

Comment:
It looks safe to uncomment/clean this.

Reply:
Yes, I was using it as a default. Removed!